### ISSUE TITLE
abnormal circulating cartilage oligomeric matrix protein level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3092]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

a. abnormal circulating cartilage oligomeric matrix protein level
b. increased circulating cartilage oligomeric matrix protein level 
c. decreased circulating cartilage oligomeric matrix protein level 

>2. Synonyms (e.g., absent spleen)
a. abnormal circulating COMP
b. increased circulating COMP
c. decreased circulating COMP

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

PMID:8909256: 
a. anomaly in the amount in the blood, plasma, or serum of cartilage oligomeric matrix protein
b. higher than normal levels of cartilage oligomeric matrix protein in the blood, plasma, or serum
c. lower than normal levels of cartilage oligomeric matrix protein in the blood, plasma, or serum

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a. abnormal circulating protein level (MP:0005416)
b. abnormal circulating cartilage oligomeric matrix protein level (new term)
c. abnormal circulating cartilage oligomeric matrix protein level (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjwang

**Comment by anna-anagnostop** [comment]
Definition sources: https://www.uniprot.org/uniprot/P49747, https://en.wikipedia.org/wiki/Cartilage_oligomeric_matrix_protein

[PMID: 31963737](https://pubmed.ncbi.nlm.nih.gov/31963737/):
COMP, also referred to as thrombospondin 5, is a pentameric glycoprotein, which is expressed in cartilage and numerous other tissues/cells including ligaments, synovium, fibroblasts/myofibroblasts, vascular smooth muscle cells, cardiomyocytes, and various tumor cells [13]. It is known to be involved in collagen secretion and fibrillogenesis, and interacts with various other components of the extracellular matrix (ECM) like collagen types I, II, IX, matrillin 3, aggrecan, and fibronectin [10,14]. While mutations in the COMP gene are associated with two forms of skeletal dysplasia—pseudoachondroplasia and multiple epiphyseal dysplasia—altered expression and/or systemic release of wild type COMP has been described in a number of predominantly age-dependent and degenerative diseases such as osteoarthritis, cardiomyopathy, fibrosis as well as breast and prostate cancers [13,15]. Primarily, COMP has been considered as a quite specific indicator of cartilage ECM turnover, which is elevated during joint degeneration in osteoarthritic disease or rheumatoid arthritis but also after intense loading due to excessive physical activity or joint trauma [16,17,18] but it becomes more and more clear that various tissues and diseases contribute to its serum level.

**Comment by sbello** [comment]
MP:0021034 - abnormal
MP:0021035 - decreased
MP:0021036 - increased
